docs/2-building-blocks-of-oop-part-1/hometask/src/Point.ts

```diff
@@ -1 +1,39 @@
-export class Point {}
+export class Point {
+  private x: number;
+  private y: number;
+
+  private calculateDistanceTo(x: number, y: number): number {
+    const distance: number = Math.sqrt(
```

Comment:
Pls take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot

Just for you to know )

Reply:
thank you  )